vetiver/templates/monitoring_dashboard.qmd

```diff
@@ -0,0 +1,138 @@
+---
+title: "Monitoring dashboard"
+format:
+  dashboard:
+    orientation: columns
+logo: https://github.com/rstudio/vetiver-python/blob/main/docs/figures/logo.png?raw=true
+output: asis
+jupyter: python3
+---
+
+```{python}
+#| include: false
+#| tags: [parameters]
+
+# import model and metadata
+import pins
+from IPython.display import display, Markdown, IFrame
+from datetime import datetime, timedelta
+import pandas as pd
+import plotly.express as px
+from sklearn import metrics
+from vetiver import VetiverModel, compute_metrics, plot_metrics
+from sklearn.metrics import recall_score, accuracy_score
+
+raw = "https://colorado.rstudio.com/rsc"
+paths = {"chicago-model-python": "chicago-model-python/"}
+board = pins.board_url(raw, paths, allow_pickle_read=True)
+v = VetiverModel.from_pin(board, "chicago-model-python")
+v_meta = board.pin_meta("chicago-model-python")
+days_old = datetime.today() - datetime.strptime(v_meta.created, "%Y%m%dT%H%M%SZ")
+```
+
+```{python}
+## the next few lines are an example model, here is a place to
```

Comment:
This might be a little heavyweight to just load in monitoring data, but I wonder if something like mtcars is too simple?

Reply:
I do think something like mtcars is too simple because it doesn't have dates. This seems pretty reasonable, although some kind of built-in data would be great too.